code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -98,6 +98,7 @@
 	air_update_turf(FALSE, FALSE)
 	var/static/list/loc_connections = list(
 		COMSIG_ATOM_ENTERED = .proc/on_entered,
+		COMSIG_ATOM_ABSTRACT_ENTERED = .proc/on_entered,
```

Comment:
even with the tram we are still entering the bounds of the turf and thus touch the air, so hotspots should register to things entering

Reply:
?